internal/proto/controller/api/resources/hosts/v1/host.proto

```diff
@@ -51,20 +51,20 @@ message Host {
   ]; // @gotags: `class:"public"`
```

Comment:
Are we adding PluginInfo separately, or is that not included?

Reply:
Yes, we are adding them inside `plugin.proto`